use mqtt v1 in docs (#2384)
thin-edge/thin-edge.io: docs/src/start/raise-alarm.md

An empty message can't be used for the same, as empty messages are used to clear alarms, which is discussed in the next section.
:::

> The `<severity>` value in the MQTT payload can only be one of the following values:

Reviewer: I would change these severities are the "recommendations".

```suggestion
The `<severity>` value in the MQTT payload is recommended to be one of the following values:
```
